configs/components/facter.rb

```diff
@@ -109,10 +109,13 @@
       skip_blkid = 'OFF'
     end
   end
+  if platform.is_huaweios?
+    skip_blkid = 'ON'
```

Comment:
Does this not default to on? Should we append `|| !platform.is_huaweios?` to line 100?